projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/MultiProgress.svelte

```diff
@@ -34,6 +34,8 @@
 					isLink={hasIssues}
 					class="jb-navigator-link"
 					to="/image-size-analysis/{group}/1"
+					trackEvent="clicked_isa_settings_group"
+					trackEventProps="group"
```

Comment:
I think this should be the group name. Maybe you meant to write `{group}` instead of `"group"` 😅